javascript-client/getting-started.md

```diff
@@ -100,12 +100,19 @@ are relayed through GSN
 ```javascript
 const { RelayProvider } = require('@opengsn/gsn')
 
-const config = { paymasterAddress }
+const config = { 
+    paymasterAddress,
+    loggerConfiguration: {
+        logLevel: 'debug',
+        // loggerUrl: 'logger.opengsn.org',
+    }
+}
 const provider = await RelayProvider.newProvider({ provider: web3.currentProvider, config }).init()
 const web3 = new Web3(provider);
 ```
 
 With these changes, your application will route the requests through GSN.
+The "loggerUrl" is optional: setting it, will send the logs to opengsn global logger, to help troubleshooting by the GSN support. 
```

Comment:
Setting it to 'logger.opengsn.org' will. Please at least mention that people can provide their own endpoint for GSN logs collection.